src/Component/CountFunc.js

```diff
@@ -0,0 +1,43 @@
+// import logo from './logo.svg';
+import './App.css';
+import React, {useState} from "react";
+
+
+function App() {
+  const [count, countocc] = useState(0); 
+
+  function addcounter() {
+    countocc(function (prevCount) {
+      return (prevCount += 1);
+    });
+  }
+
+  function decreasecounter() {
+    countocc(function (prevCount) {
+      if (prevCount > 0) {
+        return (prevCount -= 1); 
+      } else {
+        return (prevCount = 0);
+      }
+    });
```

Comment:
this is not correct way of implemeting counter increment and decrement , optimize this